packages/pluggableWidgets/datagrid-date-filter-web/src/components/FilterComponent.tsx

```diff
@@ -48,10 +48,23 @@ export function FilterComponent(props: FilterComponentProps): ReactElement {
                     ariaLabel={props.screenReaderButtonCaption}
                     defaultFilter={props.defaultFilter}
                     name={props.name}
-                    onChange={type => {
-                        setType(type);
-                        focusInput();
-                    }}
+                    onChange={useCallback(
+                        type => {
+                            setType(type);
+                            focusInput();
+                        },
+                        [focusInput]
+                    )}
+                    options={
+                        [
+                            { value: "greater", label: "Greater than" },
+                            { value: "greaterEqual", label: "Greater than or equal" },
+                            { value: "equal", label: "Equal" },
+                            { value: "notEqual", label: "Not equal" },
+                            { value: "smaller", label: "Smaller than" },
+                            { value: "smallerEqual", label: "Smaller than or equal" }
```

Comment:
Shouldn't these labels be translatable? Might be out of scope for this PR, but something to think about :)

Reply:
@cjfhodges we need to keep track of the necessity of translations for piw-apis. We had same problem for pagination.